CMakeLists.txt

```diff
@@ -28,6 +28,8 @@ set(MARABOU_TEST_LIB MarabouHelperTest)
 set(MARABOU_EXE Marabou${CMAKE_EXECUTABLE_SUFFIX})
 set(MARABOU_PY MarabouCore)
 
+
```

Comment:
Extra spaces and empty lines in this file.

Reply:
Thanks, will make spacing uniform throughout the file.